lib/driver_sht41/include/driver/sht41.h

```diff
@@ -12,21 +12,70 @@
  *
  */
 
+// i2c address for sht41 driver
+#define SHT41_I2C_ADDR 0x44
+
+// time needed for sht41 driver to complete sampling temperature and humidity
+#define SHT41_SAMPLING_TIME_MS 15
```

Comment:
For parameters which only make sense in the context of the c file, keep them in there, rather than copying them into the header file. For example, the board code doesn't have a use to know what sampling time is